clustered/server/src/main/java/org/ehcache/clustered/server/offheap/OffHeapChainMap.java

```diff
@@ -43,11 +44,11 @@
   }
 
   private final ReadWriteLockedOffHeapClockCache<K, InternalChain> heads;
-  private final OffHeapChainStorageEngine<K> chainStorage;
+  private final ChainStorageEngine<K> chainStorage;
```

Comment:
hmm.. interesting point,, 

Anyway for everything there is some association to offheap...just wanted to avoid connecting to a concrete class for extensibility and ability to connect different storage engines to the chain map.
OffHeapChainStorageEngine is a concrete class..while ChainStorageEngine is an interface.